### PR TITLE
Adds divs and some styling for creating ribbon on screen reduction

### PR DESCRIPTION
**Pull request by kathyoleary**
- [x] Added divs for LMHT and Contribute Ribbon on screen reduction.
- [x] Finish styling in CSS to get it to pop the logo and Contribute Ribbon in the divs created on screen reduction.
- [x] Test thoroughly!  Have a double check on the coding, some should be conflicting, but its working so I'm leaving it in for now.

### CODE REVIEW

**Comment by kathyoleary** [comment]
:green_heart: Ready to go!  Thumbs please!

**Comment by ericanaughton** [comment]
Great work @kathyoleary! Some things I noticed:
1. The ribbon is a bit jagged on the bottom left side
image
2. It'd be nice if it had a hover state similar to the other ribbon.
3. When I click on the new ribbon, it brings me to the Welcome page, when I expect to be directed to the Contribute page on Github. 
4. Please ABC order your CSS properties. You can install a package in Atom called Sort. Then all you need to is press F5 when you've highlighted the area you want sorted :)

**Comment by kathyoleary** [comment]
ok question....mine was showing perfectly smooth and if I just use my mousepad to zoom in and zoom out the size it flows perfect, but once I used the drop down menu and did View - Zoom In...I now have the jagged edge and it won't go away?  (and I did refresh the screen before too and it didn't change.  Wasn't until I did the drop down zoom that it switched)  Do I just have a wonky refresh that isn't working well?

screen shot 2016-04-15 at 1 36 54 pm

**Comment by kathyoleary** [comment]
interesting point to add....if you use command + and command - this goes from the ribbon being ok to jagged to ok to jagged.....

**Comment by missingdink** [comment]
👍

File: index.html [hunk 1]
```diff
@@ -44,6 +44,9 @@
   </header>
     <div class="ui-widget">
     <nav>
+      <div class="horizontal-lmht"></div>
+      <a href="https://github.com/LMHT/LMHT.github.io/blob/master/CONTRIBUTING.md" target="_blank" class="horizontal-ribbon"><img width="40" src="img/github.svg"><div>Contribute</div>
```
Comment (werbeckes) [inline index.html]: Consider making the image and div tags on their own line.  This one is pretty long.

**Comment by werbeckes** [comment]
👍 
Plus one comment.

**Comment by kathyoleary** [comment]
Adjusted line length to finish last thumbs up.

**Pull request merged**